src/main/scala/firrtl/passes/InferTypes.scala

```diff
@@ -110,7 +110,7 @@ object CInferTypes extends Pass {
 
     def infer_types_e(types: TypeMap)(e: Expression) : Expression =
       e map infer_types_e(types) match {
-         case (e: Reference) => e copy (tpe = (types getOrElse (e.name, UnknownType)))
+         case (e: Reference) => e copy (tpe = types getOrElse(e.name, UnknownType))
```

Comment:
getOrElse isn't arity-1..

If you wanted to chain arity-1 can do get(key): Option[B] and then getOrElse on that option.